sdk/cosmosdb/cosmos/src/queryExecutionContext/orderByComparator.ts

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { NonStreamingOrderByResult } from "./nonStreamingOrderByResult";
+
+/** @hidden */
+const TYPEORDCOMPARATOR: {
+  [type: string]: { ord: number; compFunc?: (a: any, b: any) => number };
```

Comment:
Please add comments explaining the type. For example, explain `ord` field, how it is used? It seems it is used to figure out if operators are of same type, but let's document its usage properly here.

Reply:
I will create a new PR addressing this issue https://github.com/Azure/azure-sdk-for-js/issues/30122.